code-challenges.test.js

```diff
@@ -12,35 +12,99 @@
 
 // --------------------1) Create a function that takes in an array of objects and returns an array with a sentence about each person with their name capitalized.
 
+// Inputs: array of Objects
+// Output: undefined
+// Parameters: variable object
+
 // a) Create a test with an expect statement using the variable provided.
 
-const people = [
-  { name: "ford prefect", occupation: "a hitchhiker" },
-  { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-  { name: "arthur dent", occupation: "a radio employee" }
-]
+// describe("capitalizer", () => {
+//   it("returns an array with a sentence about each person with their name capitalized", () => {
+
+//   const people = [
+//     { name: "ford prefect", occupation: "a hitchhiker" },
+//     { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
+//     { name: "arthur dent", occupation: "a radio employee" }
+//   ]
+//     expect(capitalizer()).toEqual('["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]')
+//      })
+//    })
+
+// const capitalizer = (obj) => {
+//   // takes in an object and iterate over each value
+//   return Object.name(n).splice(0, 1).toupperCase().join(" ")
+//   // values at the first index should be captilalized and joined.
+// }
+
 // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
 
 // b) Create the function that makes the test pass.
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+
 
 // --------------------2) Create a function that takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.
 
+// Inputs: array 
+// Output: undefined
+// Parameters: array of mixed data
+
 // a) Create a test with an expect statement using the variables provided.
 
-const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
-// Expected output: [ 2, 0, -1, 0 ]
-const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
-// Expected output: [ 2, 1, -1 ]
+//  describe("mxdtrtn", () => {
+//    it("returns a generic greeting", () => {
```

Comment:
Make sure to make the description in the it section descriptive of your intent for the test.